package.json

```diff
@@ -9,7 +9,7 @@
     "test": "npm run test-only",
     "check": "run-s lint test",
     "clean": "lerna run --parallel clean && lerna clean --yes && rimraf node_modules",
-    "build": "lerna run --parallel build",
```

Comment:
was this causing a problem? 🤔

Reply:
when I added  a build step for  `saga-monitor-example` running `npm run build` started failing. from `lerna` [docs](https://github.com/lerna/lerna/tree/master/commands/exec#--parallel)
```
--parallel
...
Similar to --stream, but completely disregards concurrency and topological sorting,
```